src/transformer/utils/constants.php

```diff
@@ -2,7 +2,7 @@
 
 namespace transformer\utils;
 
-const xapi_type = 'http://id.tincanapi.com/activitytype/lms';
+const xapi_type = 'http://id.tincanapi.com/activitytype/lms/';
```

Comment:
why is this added?

Reply:
Just temporary until we create a different URL for each type.

Reply:
"there's nothing as permanent as a temporary solution"